audited.gemspec

```diff
@@ -16,11 +16,11 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ">= 2.3.0"
 
-  gem.add_dependency "activerecord", ">= 5.2", "< 7.2"
-  gem.add_dependency "activesupport", ">= 5.2", "< 7.2"
+  gem.add_dependency "activerecord", ">= 5.2", "< 8.1"
```

Comment:
@danielmorrison setting `< 8.1` is the same as supporting rails `main` branch, which is "edge rails"

So why not just removing the upper bound like so `gem.add_dependency "activerecord", ">= 5.2"` so that PRs like this won't be necessary, and future PRs by contributors will be about fixing bugs on newer versions of rails, instead of these "Bump rails " kind of PRs